packages/ember-routing/lib/services/router.js

```diff
@@ -62,8 +62,24 @@ const RouterService = Service.extend({
        attempted transition
      @public
    */
-  replaceWith() {
+  replaceWith(/* routeNameOrUrl, ...models, options */) {
     return this.router.replaceWith(...arguments);
+  },
+
+  /**
+     Generate a URL based on the supplied route name.
+
+     @method urlFor
+     @param {String} routeName the name of the route
+     @param {...Object} models the model(s) or identifier(s) to be used while
+       transitioning to the route.
+     @param {Object} [options] optional hash with a queryParams property
```

Comment:
Yeah, I think it makes sense to use the same interface as `.transitionTo` / `.replaceWith` (which both have options with a `queryParams` property on them).  We can get @ef4 and others to confirm just before landing, but I think it makes sense to stay consistent for now.

If we do decide to change the API, they should all be changed, which is definitely unrelated to this PR...